contrib/FreeBSD/README.md

```diff
@@ -0,0 +1,58 @@
+This folder contains `sopel` and `sopel-default.cfg` designed to be distributed by third parties such as FreeBSD.
+
+`sopel-default.cfg` is a default configuration file for sopel.
+
+The -default prefix is mandatory and will be used as a 'profile'.
+
+The profiles may be changed with the variable `sopel_profiles` in `/etc/rc.conf`. For example:
+
+```sh
+sysrc sopel_profiles="profile1 profile2 profile3"
+```
+
+By default, the configuration directory is `/usr/local/etc`.
+
+The `sopel-profile1.cfg`, `sopel-profile2.cfg` and `sopel-profile3.cfg` files must be stored in the configuration directory.
+
+sopel- is mandatory and can be changed with the `sopel_prefix` variable in `/etc/rc.conf` as another FreeBSD service.
```

Comment:
I will remove the `sopel_prefix`, it is intended for packagers, but I suggest that `sopel_interpreter` should be kept because, on FreeBSD, there is no `/usr/local/bin/python` link to `/usr/local/bin/python3.8|3.9|x.xx`, so it is necessary to reconfigure `sopel_interpreter` when python has a new version on FreeBSD. The `sopel_interpreter` variable is not only used to run sopel, it is also used to check if sopel is running with the `check_pidfile` function.